.github/workflows/prettier-html.yml

```diff
@@ -0,0 +1,37 @@
+name: Pretty HTML
+
+on:
+  workflow_dispatch:
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout gh-pages branch
+        uses: actions/checkout@v3
+        with:
+          ref: gh-pages
+
+      - name: Find and Remove </source> Tags
+        run:  find . -type f -name "*.html" -exec sed -i 's/<\/source>//g' {} +
+      
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+
+      - name: Install Prettier
+        run: npm install -g prettier
+
+
+      - name: Check for Prettier
+        run: npx prettier --version || echo "Prettier not found"
+
+      - name: Run Prettier on HTML files
+        run: npx prettier --write '**/*.html'
```

Comment:
Using this regex also gets the html files in the root directory right? Just to be sure, because I don't remember where in the code using `**/` only gets the files in subdirs.

Reply:
Yes it gets all of the html files in the root directory as well.
```**/``` means all folders including root and ```*.html``` means all html files